roles/slurm_client/templates/nhc.conf

```diff
@@ -87,30 +84,69 @@
 # Make sure the /var filesystem has enough free inodes.
    * || check_fs_ifree /var 1k
 
-# The following illustrates how to assert an NFSv3 mount (or any other specific mount option).
-#   * || check_fs_mount -s bluearc0:/home -t nfs -o '/(^|,)vers=3(,|$)/' -f /home
+#
+## Home
+# 
+# All UI's and compute nodes should have their home filesystem mounted read/write
+# Parsed from what pfs storage is home lfs, then in that pfs get mount type. If 'none', set to xfs/etx4.
+{% for lfsitem in lfs_mounts | selectattr('lfs', 'search', 'home') -%}
+  {%- for pfsitem in pfs_mounts -%}
+    {%- if pfsitem.pfs == lfsitem.pfs -%}
+      {%- if pfsitem.type != 'none' %}
+   * || check_fs_mount_rw -f /home -t '{{ pfsitem.type }}'
+      {%- else %}
+   * || check_fs_mount_rw -f /home -t '/(xfs|ext4)/'
+      {%- endif -%}
+    {%- endif -%}
+  {%- endfor -%}
+{%- endfor %}
```

Comment:
```suggestion
{% for lfs_item in lfs_mounts | selectattr('lfs', 'search', 'home') -%}
  {%- set fs_type = pfs_mounts | selectattr('pfs', 'match', lfs_item.pfs) | map(attribute='type') | first -%}
  {%- if fs_type != 'none' %}
   * || check_fs_mount_rw -f /home -t '{{ fs_type }}'
  {%- else %}
   * || check_fs_mount_rw -f /home -t '/(xfs|ext4)/'
  {%- endif -%}
{%- endfor %}
```

Reply:
Note:
 * The suggestion above is  untested, but should work; we use similar lookups in the tasks for the shared_storage role.
 * You can do a direct lookup of the file system type for an LFS in the pfs_mounts data structure: no extra ```for``` loop required.
 * Same thing applies to code for other file system checks below.